examples/hu.bme.mit.massif.examples.oslc/pom.xml

```diff
@@ -10,6 +10,11 @@
 			<name>Public Eclipse Lyo maven repository.</name>
 			<url>https://repo.eclipse.org/content/groups/lyo/</url>
 		</repository>
+		<repository>
```

Comment:
Actually, a more useful approach would be to simply drop all OSLC related code from master.